README.md

```diff
@@ -1,12 +1,15 @@
 # reddit-collapse-by-default
 
-All Reddit comments collapsed by default (not just top level ones), and collapsing only hides children and doesn't hide the text of the post itself.
+All Reddit comments are collapsed by default so that only top level comments are readable. Their child comments are collapsed but visible for expanding/collapsing as needed, if the top level comment is interesting enough to dive into.
+
+![image](https://user-images.githubusercontent.com/8013867/34451500-b80c29fa-ecf5-11e7-8ef7-94c0259315dc.png)
+
 
 I am trying to test a hunch that I've had for some time that it would be better to read sibling comments first before deciding to go deeper onto any child comments.
 
 Potential TODOs
 
-- [ ] Integrate with RES auto-remember feature
+- [x] Integrate with RES auto-remember feature
```

Comment:
What is the RES auto-remember feature?

Is reddit remembering what we collapsed and expanded? That's lucky!